numpy/core/_add_newdocs.py

```diff
@@ -3964,18 +3964,19 @@
     >>> x.tobytes('F')
     b'\\x00\\x00\\x02\\x00\\x01\\x00\\x03\\x00'
 
-    """
+    """))
+
+
+add_newdoc('numpy.core.multiarray', 'ndarray', ('tostring', r"""
+    a.tostring(order='C')
+
+    A compatibility alias for `tobytes`, with exactly the same behavior.
+
+    Despite its name, it returns `bytes` not `str`\ s.
+
+    .. deprecated:: 1.19.0
+    """))
```

Comment:
Do we need to include the parameters, return and example to this docstring ? I see other deprecated functions in the code base where this information is present.

Reply:
I'd argue in this case there's no point, as the functions are completely identical.

Reply:
This will still have a slight impact on users continuing to use the deprecated tostring and calling help on tostring instead of tobytes (probably just making them run an addition help on tobytes).

Reply:
Not sure, but then it is identical and not having a full documentation means that you cannot overread the "deprecated" note, so I think I am happy with it.

Tested the ma and scalars manually, so OK with this (the C-code needs the test the most). Will probably just merge later (and leave release notes formatting, if wished, for release time).